azure-pipelines/azure-pipelines.yml

```diff
@@ -468,18 +468,25 @@ stages:
           -t edgehub/mockclient:$(tag) --load
         displayName: build mockClient
       - script: |
-          docker buildx build -f --platform=linux/amd64 \
+          docker buildx build --platform=linux/amd64 \
           -f $(System.DefaultWorkingDirectory)/examples/telemetryservice/mockserver/Dockerfile.mockserver \
           --build-arg PROJECT_ROOT="$(System.DefaultWorkingDirectory)" $(System.DefaultWorkingDirectory) \
           -t edgehub/mockserver:$(tag) --load
         displayName: build mockServer
       - script: |
           set -e
           docker run -itd --network host --name mosquitto eclipse-mosquitto:2.0.14
+          docker run -itd --network host --name tdengine \
```

Comment:
Let's put TDengine's test into a separate stage. If you mix MQTT and TDengine, if the test failed, you need extra time to debug which caused the test fail.